engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -225,6 +224,10 @@ private void initPipeline() {
         renderingPipeline.add(lightGeometryNode);
         renderingPipeline.add(directionalLightsNode);
         renderingPipeline.add(chunksRefractiveReflectiveNode);
+        renderingPipeline.add(outlineNode);
+        renderingPipeline.add(ambientOcclusionPassesNode);
+        renderingPipeline.add(prePostCompositeNode);
+        renderingPipeline.add(simpleBlendMaterialsNode);
```

Comment:
This is starting to look very nice. Let's make sure it remains nice when we start adding edges.